Makefile

```diff
@@ -11,7 +11,7 @@ TIMEOUT := 60
 UNION := functional docker crio docker-compose network netmon docker-stability oci openshift kubernetes swarm vm-factory entropy ramdisk shimv2
 
 # skipped test suites for docker integration tests
-SKIP :=
+SKIP := $(shell bash -f .ci/hypervisors/$(KATA_HYPERVISOR)/filter_docker_$(KATA_HYPERVISOR).sh)
```

Comment:
hold on. what happen when KATA_HYPERVISOR is qemu? :thinking:

Reply:
http://jenkins.katacontainers.io/job/kata-containers-tests-ubuntu-16-04-PR/1108/consoleFull

Reply:
thanks @GabyCT